### PR TITLE
Move repetitive trigger with event to separate private method

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
@EmanueleMinotto I still not sure about the docblock, could you verify? Thanks ;)

File: src/App.php [hunk 1]
```diff
@@ -127,11 +124,28 @@ public function run($request = null, $response = null)
...
+     * Event Manager trigger with exception
+     *
+     * @param \Zend\EventManager\EventManager|\Cake\Event\EventManager $eventManager
+     * @param PennyEventInterface|\Zend\EventManager\EventInterface|\Cake\Event\Event $event
+     * @param string $name
+     * @param Exception $exception
+     *
+     * @return PennyEventInterface|\Zend\EventManager\EventInterface|\Cake\Event\Event
+     */
+    private function triggerWithException($eventManager, $event, $name, $exception)
```
Comment (EmanueleMinotto) [inline src/App.php]: `$exception` could be forced to be an `\Extension`

Comment (samsonasik) [inline src/App.php]: done ;)

**Comment by EmanueleMinotto** [comment]
Looks good to me :+1:

**Pull request merged**